client/webpack.config.js

```diff
@@ -293,6 +294,7 @@ const webpackConfig = {
 	plugins: [
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify( bundleEnv ),
+			'process.env.NODE_DEBUG': JSON.stringify( process.env.NODE_DEBUG || false ),
```

Comment:
This is used by `util.js`: https://github.com/browserify/node-util/blob/master/util.js#L109